### DISCUSSION TITLE
Customize display of objectbricks in editor

### DISCUSSION THREAD

**Discussion by bs-csa** [pimcore/pimcore#16939] [Q&A / Support] [answered]
Hi there,
we need the style of the objectbricks displaying in separate tabs to be changed to simple field groups.
My idea was that we can change or override a given twig template or such, but i could not manage to find these.

Is there any way to customize the editor?

Thank you for any hints.

**Comment by markus-moser** [comment, marked as answer]
This will not work - the backend is based on ExtJS so twig templates do not exist for cases like this. If you invest a lot if time in theory it could be possible to change what you would like to achieve but I do not recommend it. This should stay like it is as it would need to much customization effort.
